extensions/wikia/Recirculation/js/recirculation.js

```diff
@@ -84,9 +85,21 @@ require([
 	});
 
 	// fetch data for all recirculation modules
-	// TODO lazy load some data on scroll
 	liftigniter.fetch('ns');
-	liftigniter.fetch('wiki');
+
+	window.addEventListener('scroll', function () {
+		var mcfOffset = $mixedContentFooter.offset().top,
+			scrollPosition = $(window).scrollTop(),
+			windowInnerHeight = $(window).innerHeight(),
```

Comment:
This is pretty heavy to run on every scroll event (there's a lot of them). Consider using a throttle.